.circleci/config.yml

```diff
@@ -280,9 +281,6 @@ jobs:
       - run:
           name: 'Authenticate with registry'
           command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/repo/.npmrc
-      - run:
-          name: 'Build all packages with lerna'
-          command: 'yarn build --skip-nx-cache'
```

Comment:
Please verify that the packages all have the dist folders as expected. I recall there was a comment in the code that said without the second extra build the just folders were missing for certain packages.

Reply:
Yes, tests are running using the cashed build here 

```
> @requestnetwork/types:"lint:check"  [existing outputs match the cache, left as is]
```

```
> @requestnetwork/types:build  [local cache]
```

You can see in this logs:

https://app.circleci.com/pipelines/github/RequestNetwork/requestNetwork/10402/workflows/cbcc1fc6-436b-4349-aa0e-e0e420a4a29e/jobs/159681/parallel-runs/0/steps/0-102

So it should have it persisted to the next-release job as well.